packages/wdio-cli/src/index.js

```diff
@@ -25,6 +25,9 @@ export const run = async () => {
         .commandDir('commands')
         .example('$0 run wdio.conf.js --suite foobar', 'Run suite on testsuite "foobar"')
         .example('$0 run wdio.conf.js --spec ./tests/e2e/a.js --spec ./tests/e2e/b.js', 'Run suite on specific specs')
+        .example('$0 run wdio.conf.js --spec ./tests/e2e/a.feature:5', 'Run scenario by line number')
+        .example('$0 run wdio.conf.js --spec ./tests/e2e/a.feature:5:10', 'Run scenarios by line number')
+        .example('$0 run wdio.conf.js --spec ./tests/e2e/a.feature:5:10,./test/e2e/b.feature', 'Run scenarios by line number in single feature and another complete feature')
```

Comment:
In v5 we don't comma separate these anymore and instead call `--spec` multiple times:
```suggestion
        .example('$0 run wdio.conf.js --spec ./tests/e2e/a.feature:5:10 --spec ./test/e2e/b.feature', 'Run scenarios by line number in single feature and another complete feature')
```

Reply:
:+1:

Reply:
Done! :-)